Sortng.py.txt

```diff
@@ -0,0 +1,108 @@
+import time
+#1
```

Comment:
Co to za komentarz? clean code proszę przeczytać pierwsze 3 rozdziały przynajmniej i nie dawać komentarzy w kodzie. Twój kod to jeden wielki skrypt, nie zawierający żadnej definicji funkcji, więc z definicji powinnaś dostać 0 pkt (zadaniem było napisanie funkcji). 'How to write function python' w Google, jeśli aż takie braki.